src/components/Pivot/Pivot.tsx

```diff
@@ -42,6 +48,7 @@ export default class Pivot extends React.Component<PivotProps> {
         selectedKey={this.props.selectedKey}
         onLinkClick={this.onLinkClick}
         headersOnly={true}
+        linkSize={this.props.linkSize ? this.props.linkSize : PivotLinkSize.normal}
```

Comment:
We luck out here in that PivotLinkSize.normal is 0 - so if this prop is undefined *or* normal - we'll use normal. Explicit undefined check might make intent more clear here?

Reply:
Are you thinking it would be safer to just not pass the prop at all when its left unspecified?

Reply:
Looks like we can just pass this right though.

While Fabric internally isn't comparing against unknown, it's also defaulting to normal size, and only adding in its extra classes for large if the linkSize is explicitly === to PivotLinkSize.large

Reply:
Updated